neo4j/pom.xml

```diff
@@ -42,6 +45,25 @@
             <plugin>
                  <artifactId>maven-checkstyle-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>animal-sniffer-maven-plugin</artifactId>
```

Comment:
Animal sniffer is already configured in the root pom, why do you need it here as well?